### ISSUE TITLE
HelpMessage parameter attribute does not dispay inside VSCode terminal

### ISSUE THREAD

**Issue by anwather** [PowerShell/vscode-powershell#836]
### System Details

- Operating system name and version: Windows 10 1703

Copy / paste the below commands into the PowerShell Integrated Terminal, and paste the output here
```
C:\source\My-Scripts> code -v

1.12.2
19222cdc84ce72202478ba1cec5cb557b71163de
PS C:\source\My-Scripts> $pseditor.EditorServicesVersion


Major  Minor  Build  Revision
-----  -----  -----  --------
1      2      1      0


PS C:\source\My-Scripts> code --list-extensions --show-versions

donjayamanne.githistory@0.2.0
kube.theme-kay@0.2.0
ms-vscode.PowerShell@1.2.1
retrotheme.theme-retro@0.1.0
```

### Issue Description

When using the helpmessage parameter attribute - if I call the function in the terminal without a mandatory parameter the helpmessage cannot be displayed.

```
Function Demo-Help  {
 [CmdletBinding()]
    Param(
        [Parameter(Mandatory=$true,HelpMessage="Enter a string for computer name",Position=0)]
        [string]$ComputerName
    )
}
```

From the terminal:
```
PS C:\source\My-Scripts> Demo-Help

cmdlet Demo-Help at command pipeline position 1
Supply values for the following parameters:
ComputerName:
```

From the PowerShell CLI or ISE:
```
C:\source\My-Scripts [master ≡ +2 ~1 -0 !]> demo-help

cmdlet Demo-Help at command pipeline position 1
Supply values for the following parameters:
(Type !? for Help.)
ComputerName:
```

**Comment by anwather** [comment]
[ParameterHelp.txt](https://github.com/PowerShell/vscode-powershell/files/1056699/ParameterHelp.txt)

**Comment by pcgeek86** [comment]
@anwather This message has some serious formatting issues. Can you please fix up the Markdown?

I was able to reproduce the same behavior, where you're not prompted to use `!?` for parameter help.

Would need @rkeithhill to weigh in on why the Integrated Terminal doesn't behave the same way as PowerShell Console Host. Is this something to do with a "mode" that the PowerShell engine is launch in, in VSCode?

**Comment by daviwil** [comment]
Fixed the formatting.  Thanks for pointing this out @anwather!  Looks like this is an inconsistency in our host implementation, I'll get that fixed.

**Comment by rjmholt** [comment]
> Is there a buggy version of powershell especially made just for vscode?

This feature doesn't come from PowerShell, it comes from [PSReadLine](https://github.com/lzybkr/PSReadLine).

We have an issue tracking integration of PSReadLine in https://github.com/PowerShell/vscode-powershell/issues/535

**Comment by SydneyhSmith** [comment]
Closing as PSReadLine is now available in the PowerShell Preview extension, tracking that PSReadLine is not available in the main PowerShell Extension here #1793
